### PR TITLE
Laboratory work #1, Lopatko Yulia - 21FPL1

### PR DESCRIPTION
**Pull request by lazykosti**

### CODE REVIEW

File: lab_1_keywords_tfidf/main.py [hunk 1]
```diff
@@ -1,72 +1,62 @@
...
-    top (int): Number of token to extract
-
-    Returns:
-    List[str]: Sequence of specified length
-    consisting of tokens with the largest frequency
-
-    In case of corrupt input arguments, None is returned
-    """
-    pass
+with open("Дюймовочка.txt", "r", encoding="utf-8") as f1:
```
Comment (artyomtugaryov) [inline lab_1_keywords_tfidf/main.py:3]: Don't read anything in this file. Please, use the functions in start.py file

File: lab_1_keywords_tfidf/main.py [hunk 2]
```diff
@@ -1,72 +1,62 @@
...
+    f2 = open("stop_words.txt", "r", encoding="utf-8")
+    text = f1.read()
+    stopwords = f2.read()
+    stop_words = stopwords.split()
+    tokens = []
+    tokens_clean = []
+    frequencies = {}
+    top = 6
+
+def clean_and_tokenize(text: str):
```
Comment (artyomtugaryov) [inline lab_1_keywords_tfidf/main.py:13]: You mustn't change arguments and types of arguments or anything has already been doing for you. Revert this changes and place implementation of each function after docstring

File: seminars/practice_1_intro.py [hunk 3]
```diff
@@ -1,42 +0,0 @@
-"""
```
Comment (demid5111) [inline seminars/practice_1_intro.py:1]: you should not remove the file (!) return it back

File: requirements.txt [hunk 4]
```diff
@@ -1 +1,9 @@
+black==22.6.0
```
Comment (demid5111) [inline requirements.txt:1]: requirements.txt should not be changed as well. discard changes

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**